src/main/java/frc/robot/commands/pathplanning/BasicShoot.java

```diff
@@ -25,13 +27,19 @@ public BasicShoot(Shooter shooter, LightStrip lightStrip, double time) {
     public void initialize() {
         timer.reset();
         activated = true;
-        shooter.setShooterMotorLeftRPM(Constants.SHOOTER_MOTOR_HIGH_SPEED);
-        shooter.setShooterMotorRightRPM(Constants.SHOOTER_MOTOR_LOW_SPEED);
+        
+
+        
         timer.start();
+        startTime = Timer.getFPGATimestamp();
     }
 
     @Override
     public void execute() {
+        shooter.setShooterMotorLeftRPM(Constants.SHOOTER_MOTOR_HIGH_SPEED);
+        if (Timer.getFPGATimestamp() - startTime > Constants.SHOOTER_MOTOR_STARTUP_OFFSET){
+            shooter.setShooterMotorRightRPM(Constants.SHOOTER_MOTOR_LOW_SPEED);
+        }
```

Comment:
isFinished will terminate the command potentially before the second motor kicks in.